"...flasher flash_rootfs" instead of "..._system"

ollieparanoid opened the pull request:
### New help output
```
$ pmbootstrap flasher -h
usage: pmbootstrap flasher [-h] [--method FLASH_METHOD]
                           {boot,flash_kernel,flash_rootfs,flash_system,sideload,list_flavors,list_devices}
                           ...

positional arguments:
  {boot,flash_kernel,flash_rootfs,flash_system,sideload,list_flavors,list_devices}
    boot                boot a kernel once
    flash_kernel        flash a kernel
    flash_rootfs (flash_system)
                        flash the rootfs to a partition on the device
                        (partition layout does not get changed)
    sideload            sideload recovery zip
    list_flavors        list installed kernel flavors inside the device rootfs
                        chroot on this computer
    list_devices        show connected devices

optional arguments:
  -h, --help            show this help message and exit
  --method FLASH_METHOD
                        override flash method
```
### Details
* Change `pmbootstrap flasher flash_system` command to
  `pmbootstrap flasher flash_rootfs`
* The old command still works, but all references have been changed to
  the new command
* Remove obsolete `pmbootstrap flasher export` (that was changed to
  `pmbootstrap export` a few months ago)
* Update `README.md` and ZSH auto completion
* Change the description of the generated rootfs image (not talking
  about a system image anymore, mention that it has subpartitions)
* Better description of `pmbootstrap flasher flash_rootfs --partition`

Closes #1354.

drebrez commented on pmb/export/symlinks.py:
```diff
@@ -41,7 +41,7 @@ def symlinks(args, flavor, folder):
         "uImage-" + flavor: "Kernel, legacy u-boot image format",
         "vmlinuz-" + flavor: "Linux kernel",
-        args.device + ".img": "System partition",
+        args.device + ".img": "Rootfs with subpartitions for /boot and /",
```
@ollieparanoid I think the img has partitions not subpartitions. 
What about `Rootfs image with partitions for /boot and /`?

drebrez commented on pmb/install/_install.py:
```diff
@@ -316,13 +316,13 @@ def install_system_image(args):
-        logging.info("  Flashes the system image, that has been"
-                     " generated to your device:")
+        logging.info("* pmbootstrap flasher flash_rootfs")
+        logging.info("  Flashes the generated rootfs to your device:")
```
"Flashes the generates rootfs **image** to your device." ? (add image word)

ollieparanoid commented:
Thanks for all the reviews! I've adjusted the wording as @drebrez recommended, and updated the wiki (`git pull` straight from the MediaWiki, `sed` magic, `git push`).
